Cargo.toml

```diff
@@ -25,6 +25,8 @@ futures-util = "0.3"
 lazy_static = "1.4"
 log = "0.4"
 log-panics = { version = "2", features = ["with-backtrace"] }
+modemmanager = { git = "ssh://git@github.com/omnect/modemmanager.git", branch = "review", default-features = false }
+modemmanager-sys = { git = "ssh://git@github.com/omnect/modemmanager-sys.git", branch = "review" }
```

Comment:
we do have to make these public and then use https links, otherwise we can not build trivial via yocto

Reply:
even though the actual repository name for modem manager has capital characters imho we should at least for repo modemmanager make clear, that it is about rust bindings for modemmanager.
for go bindings I found go-modemmanager, so rs-modemmanager seems reasonable to me.
for sake of uniformity the other repo would be named rs-modemmanager-sys then.
names are just suggestions, having any part in the repo name hinting at rust bindings would be fine in my opinion.

Reply:
we had a similar discussion when we introduced our `azure-iot-sdk-sys` and `azure-iot-sdk` crates. `azure-iot-sdk-rs` already exists and it is also not proposed to add `-rs` suffix (https://rust-lang.github.io/api-guidelines/naming.html).
this said, modemmanager would be fine for me. An alternative could be omnect-modemmanager but then we are not consistent with our `azure.iot-sdk` crate.

Reply:
The dependency links have been adapted for the modemmanager crate. Building should work once the PR there has been merged.